subitem.go

```diff
@@ -47,30 +52,51 @@ type SubscriptionItemBillingThresholdsParams struct {
 // SubscriptionItemListParams is the set of parameters that can be used when listing invoice items.
 // For more details see https://stripe.com/docs/api#list_invoiceitems.
 type SubscriptionItemListParams struct {
-	ListParams   `form:"*"`
-	Subscription *string `form:"subscription"`
+	EndingBefore  *string   `form:"ending_before"`
```

Comment:
Why are we losing `ListParams` suddenly?

Reply:
Fixed.

Codegen bug. This was because SubscriptionItem has a  "list" method for SubscriptionItems, but in the API definitions also has a method for "list"ing usage record summaries. Codegen was assuming only a single "list" method and was resolving to the wrong one.